bin/dbServer.py

```diff
@@ -31,12 +31,33 @@
 from flask import Flask, request
 import json
 import logging as log
+import os
 import sys
 
+import ConfigParser
+import sqlalchemy
+from sqlalchemy.engine.url import URL
+
 from lsst.dbserv import dbREST_v0
 
 app = Flask(__name__)
 
+def initEngine():
+    config = ConfigParser.ConfigParser()
+    defaults_file = os.path.expanduser('~/.lsst/dbAuth-testLocal.txt')
+    config.readfp(open(defaults_file))
+    db_config = dict(config.items("mysql"))
+    # Translate user name
+    db_config["username"] = db_config["user"]
```

Comment:
Does this imply that `user` must be present in the defaults file, what happens if it's not there?